forem/forem
# "Empty State" for Invited Members

## What type of PR is this? (check all applicable)


## Description
This PR adds an "empty state" to the invited members table, which shows when there are no invitations pending.

## Related Tickets & Documents
- Closes #17451 

## QA Instructions, Screenshots, Recordings
To QA this PR, navigate to `/admin/invitations` and ensure that when there are no invited members, that you are shown the "empty state":
#### After:
![Screen Shot 2022-05-04 at 2 54 21 PM](https://user-images.githubusercontent.com/32834804/166824252-c008827a-91f3-4544-8461-8812147c5da7.png)

### UI accessibility concerns?
Hopefully not!

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

![John Travolta in Pulp Fiction looking around](https://media.giphy.com/media/6uGhT1O4sxpi8/giphy.gif)
